### PR TITLE
Fix/image

### PR DESCRIPTION
**Pull request by vyvydkf628**
- Add footer logo
- Add result image
- Change description style

### CODE REVIEW

File: src/static/img/footer/Ainize.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<path d="M34.3839 3.27312H24.4209C23.7333 3.27312 23.1798 2.7192 23.1798 2.031C23.1798 1.34279 23.7333 0.788879 24.4209 0.788879H34.3839C35.0716 0.788879 35.625 1.34279 35.625 2.031C35.625 2.7192 35.0716 3.27312 34.3839 3.27312Z" fill="#DADADA"/>
+<path d="M18.4667 4.06206C19.5875 4.06206 20.4962 3.15274 20.4962 2.03103C20.4962 0.909325 19.5875 0 18.4667 0C17.3459 0 16.4372 0.909325 16.4372 2.03103C16.4372 3.15274 17.3459 4.06206 18.4667 4.06206Z" fill="#DADADA"/>
+<path d="M40.355 4.06206C41.4759 4.06206 42.3845 3.15274 42.3845 2.03103C42.3845 0.909325 41.4759 0 40.355 0C39.2341 0 38.3255 0.909325 38.3255 2.03103C38.3255 3.15274 39.2341 4.06206 40.355 4.06206Z" fill="#DADADA"/>
+<path d="M29.7377 5.69026C28.3455 5.69026 27.2888 6.29453 26.4334 7.06666C26.3999 6.26096 25.729 5.62311 24.9071 5.62311C24.0685 5.62311 23.3808 6.31131 23.3808 7.15058V7.77164V16.2483H26.4334V9.50056C27.1211 8.74519 27.7082 8.30876 28.5132 8.30876C29.637 8.30876 30.0228 8.91308 30.0228 10.2894V16.2651H33.0754V9.88658C33.0921 7.28486 32.069 5.69026 29.7377 5.69026Z" fill="#DADADA"/>
+</svg>
```
Comment (minsulee2) [inline src/static/img/footer/Ainize.svg:11]: Can you explain what it is?!

File: src/components/ui/Footer.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import styled from 'styled-components'
+import ainize from '../../static/img/footer/Ainize.svg'
+import github from '../../static/img/footer/Github.svg'
```
Comment (minsulee2) [inline src/components/ui/Footer.js:4]: Please add ; in it

File: src/static/img/footer/Ainize.svg (hunk 1 above)
Comment (vyvydkf628) [reply src/static/img/footer/Ainize.svg:11]: It is just Icon :)

File: src/components/ui/Footer.js (hunk 2 above)
Comment (vyvydkf628) [reply src/components/ui/Footer.js]: Done!

**Pull request merged**